tests/kernel/proc_sys_dump.pm

```diff
@@ -0,0 +1,74 @@
+# SUSE's openQA tests
+#
+# Copyright © 2017 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+# Summary: Partially dump /sys and /proc
+# Maintainer: Richard Palethorpe <rpalethorpe@suse.com>
+
+use 5.018;
+use warnings;
+use base 'opensusebasetest';
+use testapi qw(is_serial_terminal :DEFAULT);
+use utils;
+use Time::HiRes qw(clock_gettime CLOCK_MONOTONIC);
+use File::Basename 'basename';
+use JSON;
+use serial_terminal;
+require bmwqemu;
+
+sub run {
+    my ($self)     = @_;
+    my $tar        = '/tmp/tar.xz';
```

Comment:
I guess there could be a better filename, e.g. `proc_sys_dump.tar.xz`?

Reply:
Well it gets uploaded as `proc_sys_dump-tar.xz` because of OpenQA's naming scheme. I can create the directory `/tmp/proc_sys_dump/` though and put it in there though.